source/people/index.md

```diff
@@ -25,7 +25,12 @@ date: 2022-05-05 10:44:00
 1. Yuxuan Zhu (CSE BSE)
 1. Yutong Li (Math & CS-LSA BS)
 1. Zayd Uzzaman (Greenhills School)
-1. Luoxi Meng (CSE MSE)
+1. Yile Gu (CSE MSE) 
+1. Andrew Li (CSE BSE)
+1. Haochen Zhang (CSE BSE) -->  Master’s@Rice
```

Comment:
I think we can (1) either move him to past member, or (2) use (CSE BSE → Master's @Rice) as affiliation. I prefer the former unless he's actively working on something. 

Also, wrong apostrophe. Should be '

And a missing space before @Rice.